src/core/server/saved_objects/migrationsv2/model.ts

```diff
@@ -560,11 +562,13 @@ export const model = (currentState: State, resW: ResponseType<AllActionStates>):
         return {
           ...stateP,
           controlState: 'OUTDATED_DOCUMENTS_SEARCH',
+          failedDocumentIds: [],
```

Comment:
@rudolf I'm not sure if you had something else in mind to avoid using the optional type.